pythorrent/peer_stores.py

```diff
@@ -15,7 +15,7 @@
 
 import socket
 import random
-from urlparse import urlparse
```

Comment:
To Do:
Add cross compatibility for py2 and py3

Reply:
I'll take your viewpoint if you want to retain Python2 compatibility. It wouldn't be difficult, just need to do a `try` and look for an `ImportError`. `urlparse` in Python 2 and 3 seems similar enough to be all it should need. However, my preference would be to focus entirely on Python 3 as per [expected Python 2.7 deprecation](https://pythonclock.org/).

Reply:
Ok